sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -207,6 +208,7 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
     // Returns true if the plan is supposed to be sorted.
     def isSorted(plan: LogicalPlan): Boolean = plan match {
       case _: Join | _: Aggregate | _: Generate | _: Sample | _: Distinct => false
+      case _: ShowColumnsCommand => true
```

Comment:
Personally I don't think it is odd because we just want to compare the results. Adding `ShowColumnsCommand` to sorted op looks more odd to me. cc @cloud-fan